Package.swift

```diff
@@ -27,13 +35,33 @@ let package = Package(
     .package(url: "https://github.com/apple/swift-crypto.git", from: "1.1.0"),
     .package(url: "https://github.com/JohnSundell/Splash.git", from: "0.14.0"),
     .package(url: "https://github.com/swiftwasm/WasmTransformer", .upToNextMinor(from: "0.0.2")),
+    .package(url: "https://github.com/mxcl/Path.swift.git", .exact("1.0.0")),
```

Comment:
As it's unused in this PR, I think would be best to remove it for now. Additionally, the required path utilities are already available in TSC: https://github.com/apple/swift-tools-support-core/blob/main/Sources/TSCBasic/Path.swift.

Reply:
I meant to remove Path.swift for now. Was thinking of using that to respond to the open issue about using hard coded path specifiers. Then I got distracted by testing...